portal-backend/depmap/data_explorer_2/views.py

```diff
@@ -464,6 +470,64 @@ def get_labels_matching_context():
     return make_gzipped_json_response(labels_matching_context)
 
 
+@blueprint.route("/v2/context", methods=["POST"])
+@csrf_protect.exempt
+def evaluate_v2_context():
+    """
+    Get the full list of labels (in any dataset) which match the given context.
+    """
+    inputs = request.get_json()
+    context = inputs["context"]
+    dimension_type = context.get("dimension_type")
+    if dimension_type is None:
+        abort(400, "v2 Contexts must have a 'dimension_type' field")
```

Comment:
@rcreasi these couple of lines differ from the proposal you put together so I wanted to call them out and get your opinion. The old context requests had a field called `"context_type"` which really stores what breadbox calls the "dimension type". My thinking was that:
1. this might be a good opportunity to rename a confusing field name
2. changing this gives us an easy way to check "is this in the old or new format?"

If you have opposition to it, let me know and I'd be happy to change it back 👍

Reply:
I'm sure this will come back to bite me, just because the old name exists in a lot of places, but I think it makes sense rename it. They are truly synonyms -- a `context_type` isn't anything special. It's there to make sure it's compatible with the dimension you're working with so it should've always been called `dimension_type`.

Reply:
The only reason I could think of to leave the name distinct is that the `dimension_type` might be `"screen"` and it could theoretically  make sense (though there is no support for this kind of thing today) to apply a `"depmap_model"` context against it.

Reply:
But even then that would be two dimension types that have a special relationship. So there's really no reason to call it `context_type`.